pkg/appconsts/app_consts.go

```diff
@@ -3,9 +3,10 @@ package appconsts
 import "time"
 
 const (
+	// Version is the current application version.
 	Version uint64 = 5
 	// SquareSizeUpperBound imposes an upper bound on the max effective square size.
-	SquareSizeUpperBound int = 128
+	SquareSizeUpperBound int = 512
```

Comment:
this means we no longer need the talis specific one. Or we could increase it to 1024